### PR TITLE
Action link localization

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

**Comment by hishamco** [comment]
I need to rebase once the #824 is merged

File: Oqtane.Client/Modules/Controls/ActionLink.razor [hunk 1]
```diff
@@ -93,6 +95,7 @@
 
         }
 
+        _text = Localize(nameof(Text));
```
Comment (hishamco) [inline Oqtane.Client/Modules/Controls/ActionLink.razor:98]: See how things got simpler after `LocalizableComponent` ;)

**Comment by sbwalker** [comment]
@hishamco ActionLink is broken after this PR.

image

**Comment by hishamco** [comment]
Of course, coz you merge this early, I was waiting to merge the localizable component, so I can rebase. If you remember I added an extra commit with out react to the changes

I will fix this now ...

**Comment by sbwalker** [comment]
Oops... my apologies!

**Comment by hishamco** [comment]
No problem, I will send a PR soon, and continue with other components

**Pull request merged**